net/gce.sh

```diff
@@ -151,6 +152,9 @@ Manage testnet instances
                       (by default preemptible instances are used to reduce
                       cost).  Note that the bootstrap leader, archiver,
                       blockstreamer and client nodes are always dedicated.
+   --self-destruct-minutes [number]
```

Comment:
Should the units here be hours?   I'd expect that'd feel more natural

Reply:
Sure!  Minutes was nice for building out the feature 😉